### ISSUE TITLE
Get MSBuild Verbosity enum from string

### ISSUE THREAD

**Issue by jozefizso** [cake-build/cake#1994]
## Feature request

I should be able to convert string value of the `verbosity` parameter to the MSBuild Verbosity enum to be able to call `MSBuild.SetVerbosity()` successfully.

`MSBuildVerbosityExtensions` class should have method opposite to the `GetMSBuildVerbosityName()` which will convert string to the `Verbosity` enum value.

### What You Are Seeing?
Error when calling `MSBuild.SetVerbosity()` with a string parameter.

### What is Expected?
I should be able to convert string value of the `verbosity` parameter to the MSBuild Verbosity enum to be able to call `MSBuild.SetVerbosity()` successfully.

### What version of Cake are you using?
0.24.0

### Are you running on a 32 or 64 bit system?
64 bit

### What environment are you running on?  Windows? Linux? Mac?
Windows

### Are you running on a CI Server?  If so, which one?
Yes, AppVeyor.

### How Did You Get This To Happen? (Steps to Reproduce)

Sample `build.cake`:

```csharp
var verbosity = Argument("verbosity", "Minimal");

Task("Build")
  .Does(() =>
  {
    MSBuild("Solution.sln", settings => settings
      .SetVerbosity(verbosity));
  });
```

### Can you point us to a project where this problem occurs?
https://github.com/NetOfficeFw/Samples

### Output Log
https://gist.github.com/jozefizso/87807f1a70d51ec24ee5a5feffc25bf5

**Comment by jozefizso** [comment]
Implementation proposed in pull request #1985

**Comment by devlead** [comment]
Fixed by #1985
